README.md

```diff
@@ -1394,6 +1394,7 @@ Table of Contents
    * [Eclipse Che](https://www.eclipse.org/che/) - Web based and Kubernetes-Native IDE for Developer Teams with multi-language support. Open Source and community driven. A online instance hosted by Red Hat is available at [workspaces.openshift.com](https://workspaces.openshift.com/).
    * [fakejson.com](https://fakejson.com/) — FakeJSON helps you quickly generate fake data using its API. Make an API request describing what you want and how you want it. The API returns it all in JSON. Speed up the go to market process for ideas and fake it till you make it.
    * [gitpod.io](https://www.gitpod.io) — Instant, ready-to-code dev environments for GitHub projects. Free for open source.
+   * [GitHub.dev](https://github.dev) Visual Studio Code in your Browser with git support.
```

Comment:
Please add this to the existing github entry that covers all the services they offer, we cannot list services multiple times the list will be thousands of lines long.